lib/grammers-client/src/utils.rs

```diff
@@ -27,7 +27,9 @@ pub(crate) fn generate_random_id() -> i64 {
             .expect("system time is before epoch")
             .as_nanos() as i64;
 
-        drop(LAST_ID.compare_exchange(0, now, Ordering::SeqCst, Ordering::SeqCst));
+        LAST_ID
+            .compare_exchange(0, now, Ordering::SeqCst, Ordering::SeqCst)
+            .unwrap();
```

Comment:
This is semantically different, but I guess in this case it's always going to be `Ok`.